bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableRowFilterConfiguration.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.dataflow;
+
+import com.google.bigtable.v1.RowFilter;
+
+/**
+ * This class encapsulates the metadata required to create a connection to a Bigtable cluster,
+ * and a specific table therein along with a filter on the table in the form of a {@link Scan}.
+ * NOTE: {@link Scan} isn't Serializable, so this class uses HBase's {@link ProtobufUtil} as part
+ * of the {@Externalizable} implementation.
+ */
+public class CloudBigtableRowFilterConfiguration extends CloudBigtableTableConfiguration {
```

Comment:
I'll remove this.  This was used as part of the CloudBigtableGrpcIO class, and should not be exposed here.